modules/runners/lambdas/runners/src/scale-runners/scale-down.ts

```diff
@@ -116,6 +116,8 @@ async function evaluateAndRemoveRunners(
           if (idleCounter > 0) {
             idleCounter--;
             console.debug(`Runner '${ec2runner.instanceId}' will kept idle.`);
+            const instanceIndex = ec2Runners.findIndex((runner) => runner.instanceId === ec2runner.instanceId);
+            ec2Runners.splice(instanceIndex, 1);
```

Comment:
Can we consolidate the gathering of `instanceIndex` and `splice`ing here to avoid code duplication?

Reply:
I was able to consolidate `findIndex`.